alphagov/paas-bootstrap
# [#161184357]  Use Terraform to create ACM certs.

What
----

This moves the ACM cert management into Terraform because it's much more robust than our set of scripts.

As part of this change, we no longer generate the apps cert in this
repo. It's only needed by the CF deployment, so doesn't make sense to
include here - we'll manage it with Terraform in the paas-cf repo. It
was originally added here to avoid having to duplicate the scripts into
paas-cf.

This additionally includes a temporary commit to import the existing cert into terraform to prevent it attempting to create a second one.

How to review
-------------

* Run the pipeline from this branch
* Verify terraform updates correctly and doesn't create a second Cert.
* Run the pipeline a second time to verify the import process is idempotent.

Who can review
--------------

Not me.
